### PR TITLE
Add doc to remove disks

### PR DESCRIPTION
**Pull request by bk201**
The doc suggests the user evict the Longhorn disk and then remove the disk in Harvester.

**Related issues**
https://github.com/harvester/harvester/issues/3345

**Preview**
https://deploy-preview-289--harvester-preview.netlify.app/dev/host/#remove-disks

### CODE REVIEW

File: docs/host/host.md [hunk 1]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+1. Go to the **Hosts** page.
+2. On the node containing the disk. Click the node name and the **Storage** tab.
+3. Find the disk to remove. Let's assume we want to remove `/dev/sdb`, and the disk's mount point is `/var/lib/harvester/extra-disks/1b805b97eb5aa724e6be30cbdb373d04`.
+
+![Find disk to remove](/img/v1.2/host/remove-disks-harvester-find-disk.png)
+
+#### Evict replicas (Longhorn dashboard)
+1. Please follow [this session](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards) to enable the embedded Longhorn dashboard.
+2. Visit the Longhorn dashboard and go to the **Node** page.
+3. Expand the node containing the disk. Check that the removing disk's mount point `/var/lib/harvester/extra-disks/1b805b97eb5aa724e6be30cbdb373d04` is in the disks list.
```
Comment (LucasSaintarbor) [inline docs/host/host.md:136]:
```suggestion
3. Expand the node containing the disk. Confirm the mount point `/var/lib/harvester/extra-disks/1b805b97eb5aa724e6be30cbdb373d04` is in the disks list. 
```

File: docs/host/host.md [hunk 2]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+![Find disk to remove](/img/v1.2/host/remove-disks-harvester-find-disk.png)
+
+#### Evict replicas (Longhorn dashboard)
+1. Please follow [this session](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards) to enable the embedded Longhorn dashboard.
+2. Visit the Longhorn dashboard and go to the **Node** page.
+3. Expand the node containing the disk. Check that the removing disk's mount point `/var/lib/harvester/extra-disks/1b805b97eb5aa724e6be30cbdb373d04` is in the disks list. 
+
+![Check the removing disk](/img/v1.2/host/remove-disks-longhorn-nodes.png)
+
+4. Click the **Edit node and disks** button; a dialog will pop up.
```
Comment (LucasSaintarbor) [inline docs/host/host.md:140]:
```suggestion
4. Select **Edit node and disks**.
```

File: docs/host/host.md [hunk 3]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+
+![Edit node and disks](/img/v1.2/host/remove-disks-longhorn-nodes-edit.png)
+
+5. Scroll to the disk. Set `Scheduling` to `Disable` and `Eviction Requested` to `True`, then click the **Save** button (Do not click the delete icon).
```
Comment (LucasSaintarbor) [inline docs/host/host.md:144]:
```suggestion
5. Scroll to the disk you want to remove. 
- Set `Scheduling` to `Disable` 
- Set `Eviction Requested` to `True` 
Select **Save**. Do not select the delete icon.
```

File: docs/host/host.md [hunk 4]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+
+![Evict disk](/img/v1.2/host/remove-disks-longhorn-nodes-evict-disk.png)
+
+6. The disk will be disabled, and please wait until the disk replica count becomes `0`.
```
Comment (LucasSaintarbor) [inline docs/host/host.md:148]:
```suggestion
6. The disk will be disabled. Please wait until the disk replica count becomes `0` to proceed with removing the disk. 
```

File: docs/host/host.md [hunk 5]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+
+![Wait replicas](/img/v1.2/host/remove-disks-longhorn-wait-replicas.png)
+
+#### Remove the disk (Harvester dashboard)
+1. Go to the **Hosts** page.
+2. On the node containing the disk, click **⋮ > Edit Config**.
```
Comment (LucasSaintarbor) [inline docs/host/host.md:154]:
```suggestion
2. On the node containing the disk, select **⋮ > Edit Config**.
```

File: docs/host/host.md [hunk 6]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+3. Click the **Storage** tab and remove the disk by clicking the **x** button.
```
Comment (LucasSaintarbor) [inline docs/host/host.md:155]:
```suggestion
3. Go to the **Storage** tab and select **x**  to remove the disk.
```

File: docs/host/host.md [hunk 7]
```diff
@@ -119,6 +119,46 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
...
+
+![Remove disk](/img/v1.2/host/remove-disks-harvester-remove.png)
+
+4. Click the **Save** button to remove the disk.
```
Comment (LucasSaintarbor) [inline docs/host/host.md:159]:
```suggestion
4. Select **Save** to remove the disk.
```

**Review by LucasSaintarbor — COMMENTED** [review]
Made some minor changes based on style guide. LGTM 👍

**Comment by bk201** [comment]
Thanks @LucasSaintarbor, accept all suggestions to a commit.

File: docs/host/host.md [hunk 8]
```diff
@@ -119,6 +119,49 @@ When multiple tags are specified for a volume, the disk and the nodes (that the
 
 :::
 
+### Remove disks
+
+Before removing a disk, you must first evict Longhorn replicas on the disk.
```
Comment (Vicente-Cheng) [inline docs/host/host.md:124]:
```suggestion
Before removing a disk, you must first evict Longhorn replicas on the disk.

:::note

The replica data would be rebuilt to another disk automatically to keep the high availability.

:::
```

**Review by LucasSaintarbor — APPROVED** [review]
Closing this review out. LGTM

**Comment by bk201** [comment]
@guangbochen I squash commits and add the same content to v1.1. Please help review again.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**